tee-worker/omni-executor/rpc-server/src/error_code.rs

```diff
@@ -15,11 +15,17 @@ const AUTH_TOKEN_CREATION_FAILED_CODE: i32 = -32007;
 const INVALID_MEMBER_IDENTITY_CODE: i32 = -32008;
 const VALIDATION_DATA_VERIFICATION_FAILED_CODE: i32 = -32009;
 const UNSUPPORTED_IDENTITY_TYPE_CODE: i32 = -32010;
-const PUMPX_API_GOOGLE_CODE_VERIFICATION_FAILED_CODE: i32 = -32011;
-const PUMPX_API_USER_CONNECTION_FAILED_CODE: i32 = -32012;
-const PUMPX_API_ERROR_CODE: i32 = -32013;
-const PUMPX_API_ADD_WALLET_FAILED_CODE: i32 = -32014;
-const INTENT_NONCE_MISMATCH_ERROR_CODE: i32 = -32015;
+
+const PUMPX_API_GOOGLE_CODE_VERIFICATION_FAILED_CODE: i32 = -32031;
```

Comment:
In my opinion this leaks implementation details -> does it make sense to make api consumer to be aware of this ? Do we expect it will conditionally make any actions based on these error codes ? If not why bother.

Reply:
We verify the result and error out if it's false - at the same time we report back the error code for it.

IMO F/E needs such information - they probabaly display it to user

Reply:
Thank you. good point. I will take this suggestion and remove it.